internal/clusters_test.go

```diff
@@ -38,3 +41,22 @@ func TestClusterCreateErrorWhenNoArguments(t *testing.T) {
 	_, _, err := RequireErrorRun(t, "clusters", "create")
 	assert.Contains(t, err.Error(), "accepts 1 arg(s), received 0")
 }
+
+// findValidClusterID lists clusters in the workspace to find a valid cluster ID.
+func findValidClusterID(t *testing.T) string {
+	ctx, wt := acc.WorkspaceTest(t)
+	it := wt.W.Clusters.List(ctx, compute.ListClustersRequest{
```

Comment:
Where is this cluster created?

Could this API return top suitable clusters? (similar to how clusterId was not set before).

Reply:
It could return >1 if we need to, but it's only used for the `clusters get` test here, so we need just 1.